test/e2e/integration/001-hello-gutenberg.js

```diff
@@ -8,4 +8,30 @@ describe( 'Hello Gutenberg', () => {
 		cy.url().should( 'include', 'post-new.php' );
 		cy.get( '[placeholder="Add title"]' ).should( 'exist' );
 	} );
+
+	it( 'Should have no history', () => {
```

Comment:
I was afraid this was a false positive because I thought Cypress would remove all popovers but confirmed it's breaking correctly when needed.